Source/WebCore/Modules/mediasource/SourceBuffer.cpp

```diff
@@ -503,7 +503,7 @@ ExceptionOr<void> SourceBuffer::appendBufferInternal(const unsigned char* data,
     if (isRemoved() || m_updating)
         return Exception { InvalidStateError };
 
-    DEBUG_LOG(LOGIDENTIFIER, "size = ", size, ", buffered = ", m_private->buffered());
+    ALWAYS_LOG(LOGIDENTIFIER, "size = ", size, ", buffered = ", m_private->buffered(), " streaming = ", m_source->streaming());
```

Comment:
This is going to generate a lot of logging, are you sure you want ALWAYS?

Reply:
The completion of appendBuffer is already always. 
so it doesn’t add that much logging: it just now have a match for begin and end.